cpmonitor/management/commands/cleanupimages.py

```diff
@@ -0,0 +1,212 @@
+import os
+import re
+
+from django.apps import apps
+from django.core.management.base import BaseCommand, CommandError
+from django.core.exceptions import ValidationError
+from django.core.validators import URLValidator
+from django.db import models as DjangoModels
+
+APP_NAME = __package__.split(".")[0]
+BASE_PATH = os.environ.get("BASE_PATH")
+IMAGES_PATH = os.environ.get("IMAGES_PATH")
+
+
+class Command(BaseCommand):
+    help = "Cleans up orphan images that are no longer used but are still linked in Markdown or file fields"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--dry",
+            action="store_true",
+            help="Do not clean up images but output a list of images which will be cleaned up",
+        )
+
+    def handle(self, *args, **options):
```

Comment:
Although it is only a cleanup job I would like to move the code into different classes like e.g. ReferencedImagesFetcher, SavedImagesResolver and call them here in the command to make the code less complex. But this would lead to classes in our domain layer. How do you think about it?

Reply:
Different classes sounds good. 
Could we add them as the _private.py module shown here: https://docs.djangoproject.com/en/4.2/howto/custom-management-commands/? Or is there another way to make clear that they belong to the new command?

Reply:
Actually now that I read all of it. I think it is not that much code and it is nicely structured. So I wouldn't put too much effort into it.

Reply:
So let it as it is?